code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -318,6 +318,9 @@
 	if(istype(card.loc, /mob/living/bot))
 		to_chat(src, SPAN_WARNING("You cannot unfold while inside the bot!"))
 		return FALSE
+	if(istype(card.loc, /obj/item/glass_jar))
+		to_chat(src, SPAN_WARNING("You cannot unfold while inside the jar!"))
+		return FALSE
```

Comment:
Just put this in the check up there.

Also, this is mildly shitcodey, but whatever.